v3/src/components/graph/components/droppable-svg.scss

```diff
@@ -11,3 +11,33 @@
     background: yellow;
   }
 }
+
+.drop-hint {
+  position:absolute;
+  top: 10px;
+  left: 0px; // calculated on fly
+  z-index:500;
+  background-color:#5b616b; // NOTE: http://web-accessibility.carnegiemuseums.org/design/color/
```

Comment:
This does not match the original exactly - I thought the contrast looked a little to subtle for accessibility purposes, so found a combination that was close but had enough contrast (according to the source, obviously need to verify)

Reply:
I agree. The contrast is better.